roles/swift-common/meta/main.yml

```diff
@@ -22,5 +22,11 @@ dependencies:
     when: openstack_install_method == 'package'
   - role: openstack-distro
     project_name: swift
-    project_packages: "{{ swift.distro.project_packages }}"
+    dependent_packages:
+      - git
```

Comment:
why are we installing git? We should only support swifttool_method: pip for centos and rhosp.